html/about.html

```diff
@@ -0,0 +1,168 @@
+        </nav>
+      </div>
+    </header>
+
+    <section class="first-section">
+      <div class="first-section-container">
+        <h2 class="open-intro-heading">"Hello! Connect, Learn & Grow"</h2>
+        <h1 class="open-heading">
+          CEOs International Community Global Summit 2021
+        </h1>
+        <div class="first-section-para">
+          <p>
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years. This year, we hope
+            to expand our invitation list including organizations and
+            invidivuals who want to work with us on shared projects that advance
+            the cause of the Commons, free culture and open knowledge. So if
+            you're active and engaged in the worlds of open content and
+            knowledge - free software advocates, Wikipedians. Open Knowledge,
+            galleries, libraries, museums, archives, governments and
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together.
+          </p>
+        </div>
+        <div class="event-details">
+          <p>
+            Please contact us for any further questions about CEOs International
+            Community Global Summit 2021!
+          </p>
+          <a href=""
+            ><p class="email-address">cicglobalsummit2021@gmail.com</p></a
+          >
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="flex-content">
+        <h2>CEOs International Community Global Summit 2021 Logo</h2>
+        <hr />
+        <p class="mobile-para">
+          The logo of CIC Global Summit 2021 was decided through the logo
+          competition from 8 June to 7 July.
+        </p>
+        <p class="desktop-para">
+          The logo of CIC Global Summit 2021 was decided through the logo
+          competition from 8 June to 7 July. After online public voting and
+          Selection Committee's judging process, the logo from Naresh Agrawal
+          won the prize. After consultation with Mr. Agrawal we optimized the
+          prize-winning-logo for this year's summit as follows:
+        </p>
+      </div>
+      <div class="logo-container">
+        <img src="/assets/cic_logoReal1.png" alt="logo" />
+      </div>
+      <hr class="horizontal-line" />
+    </section>
+
+    <section class="past-section">
+      <div class="past-section-container flex-past">
+        <h2>See The Past CEOs International Community Global Summits</h2>
+        <hr />
+        <p>
+          Take a look at the last two CIC Global Summits which took place in
+          Sounth Africa and in Abuja, Nigeria.
+        </p>
+        <img src="/assets/about4.jpg" alt="" />
+        <img src="/assets/about7.jpg" alt="" />
+      </div>
+    </section>
```

Comment:
- [ ] In this section on the desktop view the two bottom columns should be centered, in a horizontal line and have a gradient over them when a user hovers over the image in both mobile and desktop view. 

Clone | Original
------------ | -------------
![jh](https://user-images.githubusercontent.com/33067256/129426538-65d5237e-9f50-4f86-aaa7-7daa26ebee76.png) | ![km](https://user-images.githubusercontent.com/33067256/129426549-b868e105-43eb-4eeb-a3f3-3a4b86b6faf0.png)